libpod/define/info.go

```diff
@@ -26,26 +26,26 @@ type SecurityInfo struct {
 
 // HostInfo describes the libpod host
 type HostInfo struct {
-	Arch              string           `json:"arch"`
-	BuildahVersion    string           `json:"buildahVersion"`
-	CgroupManager     string           `json:"cgroupManager"`
-	CgroupsVersion    string           `json:"cgroupVersion"`
-	CgroupControllers []string         `json:"cgroupControllers"`
-	Conmon            *ConmonInfo      `json:"conmon"`
-	CPUs              int              `json:"cpus"`
-	CPUUtilization    *CPUUsage        `json:"cpuUtilization"`
-	DatabaseBackend   string           `json:"databaseBackend"`
-	Distribution      DistributionInfo `json:"distribution"`
-	EventLogger       string           `json:"eventLogger"`
-	Hostname          string           `json:"hostname"`
-	IDMappings        IDMappings       `json:"idMappings,omitempty"`
-	Kernel            string           `json:"kernel"`
-	LogDriver         string           `json:"logDriver"`
-	MemFree           int64            `json:"memFree"`
-	MemTotal          int64            `json:"memTotal"`
-	NetworkBackend    string           `json:"networkBackend"`
-	OCIRuntime        *OCIRuntimeInfo  `json:"ociRuntime"`
-	OS                string           `json:"os"`
+	Arch              string                 `json:"arch"`
+	BuildahVersion    string                 `json:"buildahVersion"`
+	CgroupManager     string                 `json:"cgroupManager"`
+	CgroupsVersion    string                 `json:"cgroupVersion"`
+	CgroupControllers []string               `json:"cgroupControllers"`
+	Conmon            *ConmonInfo            `json:"conmon"`
+	CPUs              int                    `json:"cpus"`
+	CPUUtilization    *CPUUsage              `json:"cpuUtilization"`
+	DatabaseBackend   string                 `json:"databaseBackend"`
+	Distribution      DistributionInfo       `json:"distribution"`
+	EventLogger       string                 `json:"eventLogger"`
+	Hostname          string                 `json:"hostname"`
+	IDMappings        IDMappings             `json:"idMappings,omitempty"`
+	Kernel            string                 `json:"kernel"`
+	LogDriver         string                 `json:"logDriver"`
+	MemFree           int64                  `json:"memFree"`
+	MemTotal          int64                  `json:"memTotal"`
+	NetworkBackend    map[string]interface{} `json:"networkBackend"`
```

Comment:
This is a breaking change and thus cannot be made, I suggest adding a new NetworkBackendInfo field.